website/pages/docs/k8s/crds/index.mdx

```diff
@@ -1,19 +1,16 @@
 ---
 layout: docs
-page_title: Consul Custom Resource Definitions (Beta)
-sidebar_title: Custom Resource Definitions <sup>(Beta)</sup>
+page_title: Consul Custom Resource Definitions
+sidebar_title: Custom Resource Definitions
 description: >-
-  Consul now supports managing configuration entries via Kubernetes Custom Resources.
+  Consul supports managing configuration entries via Kubernetes Custom Resources.
   These custom resource can be used to manage the configuration for workloads
   deployed within the cluster.
 ---
 
-# Custom Resource Definitions <sup>(beta)</sup>
+# Custom Resource Definitions
 
-## Overview
-
--> **This feature is currently in Beta.** It requires consul-helm >= 0.25.0 and
-consul >= 1.8.4.
+-> This feature requires consul-helm >= 0.26.0 and consul >= 1.8.4.
```

Comment:
Should we mention consul-k8s version requirements since it's listed as having a required version on line 72?